frontend/src/locales/zh-CN.yml

```diff
@@ -27,3 +35,54 @@ kbar:
   preferences:
     common: 偏好设置
     change-language: 更改语言…
+bbkit:
```

Comment:
OK for this PR.

@xiaoluoboding, is there a way to specify a different file to hold bbkit specific locale content?

Reply:
I also wondering if we should extract BBKit i18n messages from our business logic

Reply:
If one day we extract BBKit into a component library, we should extract it from our business logic.

Now we temporarily use the `bbkit` scope